website/docs/i18n/i18n-introduction.md

```diff
@@ -50,9 +50,9 @@ We don't provide support for:
 
 Overview of the workflow to create a translated Docusaurus website:
 
-- **Configure**: declare the default locale and alternative locales in `docusaurus.config.js`.
-- **Translate**: put the translation files at the correct file-system location.
-- **Deploy**: build and deploy your site using a single or multi-domain strategy.
+1. **Configure**: declare the default locale and alternative locales in `docusaurus.config.js`.
+2. **Translate**: put the translation files at the correct file-system location.
```

Comment:
Nit - better to use 1. throughout for ordered lists

Reply:
you mean use 1. everywhere?

Reply:
fixed